src/core/core-varlink.c

```diff
@@ -285,7 +325,8 @@ int manager_varlink_init(Manager *m) {
                         s,
                         "io.systemd.UserDatabase.GetUserRecord",  vl_method_get_user_record,
                         "io.systemd.UserDatabase.GetGroupRecord", vl_method_get_group_record,
-                        "io.systemd.UserDatabase.GetMemberships", vl_method_get_memberships);
+                        "io.systemd.UserDatabase.GetMemberships", vl_method_get_memberships,
+                        "io.system.ManagedOOM.GetManagedOOMCGroups",  vl_method_get_managed_oom_cgroup_paths);
```

Comment:
typo: Should likely be io.system**d**.ManagedOOM.GetManagedOOMCGroups (?) (same in `varlink_server_listen_address` later and a few other places)